query_select.go

```diff
@@ -783,28 +729,7 @@ func (q *SelectQuery) appendTables(fmter schema.Formatter, b []byte) (_ []byte,
 }
 
 func (q *SelectQuery) appendOrder(fmter schema.Formatter, b []byte) (_ []byte, err error) {
```

Comment:
We don't need wrap this method.

Reply:
Oh yeah, it was used for MSSQL before the structures were merged.